backend/db/migrations/20200923070907-add-pending-state.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`
+      ALTER TYPE enum_orders_payment_status ADD VALUE IF NOT EXISTS 'Pending';
```

Comment:
I skipped that state. Because, it seemed redundant with "Refunded" state. May be we can rename "Refunded" to "Rejected/Refunded".

Reply:
I think we are trying to capture 2 different state so I'd be inclined to have different statuses.
 - "Refunded" means the actually payment was refunded back to the buyer
 - "Rejected" is to represent the fact that the merchant never received the offline payment.

Also we should probably add a payment_type column to the orders table as an enum with possible values (CreditCard, PayPal, Offline, ERC20, etc...), so that we can more easily interpret the order's payment status in the front-end since different actions may apply depending on the payment type. For example, currently we support refunding via Stripe but not PayPal I believe. We could do this in a separate PR. I'm happy to help.

Reply:
Will add that `payment_type` before merging this PR. Thanks